package.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "@dataunions/monorepo",
-    "description": "Smart contracts, client and theGraph subgraph for dataunions.",
-    "private": true,
+    "description": "Smart contracts, client and The Graph subgraph for dataunions.",
     "version": "3.0.0-beta.1",
-    "author": "Streamr Network AG",
-    "license": "AGPL-3.0",
+    "author": "Data Union DAO <contact@dataunions.org>",
+    "license": "Apache-2.0",
```

Comment:
We shouldn't put a license designation on the root level to avoid misunderstandings that all the packages in the monorepo are under this license. Each package has its own license (or lacks one). Let's remove this line?

Reply:
ok